cmd/cli/main_test.go

```diff
@@ -86,22 +90,69 @@ func testUseInsecureKeyring(t *testing.T) {
 		cmd.Env = os.Environ()
 		cmd.Env = append(cmd.Env, "INSECURE_KEYRING=true")
 		out, err := cmd.Output()
-		require.Nil(t, err)
+		assert.Nil(t, err)
 		require.Equal(t, "{\"status\":\"OK\"}\n", string(out))
 
 		file, readErr := ioutil.ReadFile(security.GetPathToInsecureKeyring())
-		require.Nil(t, readErr)
+		assert.Nil(t, readErr)
 		require.NotNil(t, file)
 
 		secrets := []security.KeyringSecret{}
 		unmarshalErr := json.Unmarshal([]byte(file), &secrets)
-		require.Nil(t, unmarshalErr)
+		assert.Nil(t, unmarshalErr)
 		require.Len(t, secrets, 1)
 
 		secret := secrets[0]
-		require.Equal(t, "testuser", string(secret.Username))
+		assert.Equal(t, "testuser", string(secret.Username))
 		require.Equal(t, "seCretphrase", string(secret.Password))
 
 		os.Remove(security.GetPathToInsecureKeyring())
 	})
 }
+
+func testCreateProjectFromTemplate(t *testing.T) {
+	t.Run("cwctl project create --url <insecureTemplateRepo> --path <testDir>", func(t *testing.T) {
+		os.RemoveAll(testDir)
+		defer os.RemoveAll(testDir)
+
+		cmd := exec.Command(cwctl, "project", "create",
+			"--url="+test.InsecureGitRepo,
+			"--path="+testDir,
+		)
+		out, err := cmd.Output()
+		assert.Nil(t, err)
+		assert.Equal(t, "{\"status\":\"success\",\"projectPath\":\"./testDir\",\"result\":{\"language\":\"javascript\",\"projectType\":\"nodejs\"}}\n", string(out))
+	})
+	t.Run("cwctl project create --url <secureTemplateRepo> --path <testDir> --username <test.GoodGitHubUsername> --password <test.GoodGitHubPassword>", func(t *testing.T) {
```

Comment:
Trying to run this now with my credentials and getting a 404 error. Debugging but may help to have some more information somewhere about configuring the tests. 

Perhaps we should add some extra to the README / contribution guides about running the tests - this could go in there alongside details about running with `--short` etc..

Reply:
Are you using your GHE credentials, not GH credentials? Because the sample secure repo is GHE. GH creds would work for a private GH repo

Reply:
I changed the secure repo to a private GH repo, and used my GH credentials

Reply:
ah yes, thanks, turns out I need to make a change to get it to download private GH repos